go.mod

```diff
@@ -62,6 +62,7 @@ require (
 	github.com/aws/smithy-go v1.19.0
 	github.com/awslabs/amazon-ecr-credential-helper/ecr-login v0.0.0-20231024185945-8841054dbdb8
 	github.com/beevik/etree v1.3.0
+	github.com/bufbuild/connect-go v1.10.0
```

Comment:
I don't think this should be here.
At least, I'm not aware of any features that might require it

Reply:
I have no idea where it comes from. The branch I ran `go mod tidy` must have been dirty 🤷